src/interpreter.rs

```diff
@@ -855,7 +866,10 @@ impl<'a, 'tcx: 'a, 'arena> Interpreter<'a, 'tcx, 'arena> {
                             ty::TyTrait(_) => unimplemented!(),
                             _ => LvalueExtra::None,
                         };
-                        return Ok(Lvalue { ptr: ptr, extra: extra });
+                        return Ok(Lvalue {
+                            ptr: ptr,
+                            extra: extra,
+                        });
```

Comment:
The line is so short it wouldn't make much difference and good diif tools like GitHub's nowadays show part-of-line changes clearly with extra highlighting.